expression/builtin_other.go

```diff
@@ -145,6 +145,17 @@ func (c *inFunctionClass) getFunction(ctx sessionctx.Context, args []Expression)
 		sig.setPbCode(tipb.ScalarFuncSig_InDuration)
 	case types.ETJson:
 		sig = &builtinInJSONSig{baseBuiltinFunc: bf}
+		for i := range bf.args {
```

Comment:
Need `constructBinaryOpFunction` change too?

Reply:
Seems not, because it is already fixed by #8564